components/inputs/input-time.js

```diff
@@ -17,7 +18,7 @@ const TODAY = getToday();
 const END_OF_DAY = new Date(TODAY.year, TODAY.month, TODAY.date, 23, 59, 59);
 const INTERVALS = new Map();
 
-function getIntervalNumber(size) {
+export function getIntervalNumber(size) {
```

Comment:
This and `getDefaultTime` were needed in `input-time-range` so I chose to export them rather than move them to `dateTime` helper since they are very fairly `input-time` specific. I can move them to the helper instead if that makes more sense.